bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -66,7 +66,56 @@ spec:
         - |
           #!/bin/bash
           set -euo pipefail
+{{- if .IsNetworkTypeMigration }}
+          set +e
 
+          SDN_SUBNET=$(oc get hostsubnet ${K8S_NODE_NAME} -o jsonpath="{.subnet}")
+          echo "sdn hostsubnet:" ${SDN_SUBNET}
+          OVN_SUBNET=$(oc get node ${K8S_NODE_NAME} -o jsonpath='{.metadata.annotations.k8s\.ovn\.org/node-subnets}'| grep -o  -E '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\/[0-9]{1,3}')
```

Comment:
Would it be possible to use `jq '.default[0]'` instead of the grep?
I think the grep might not work if there are multiple node-subnets (secondary networks)

Reply:
The `jq` command is not available in the SDN image. I don't think we can support rollback with multiple node-subnets.